macros/utils/table_operations/make_temp_relation.sql

```diff
@@ -0,0 +1,9 @@
+{% macro make_temp_relation(base_relation, suffix=none) %}
```

Comment:
Is that fix relevant to all of the adapters?
Maybe we want to create a default one that uses the dbt one and one for databricks?